scene/gui/rich_text_label.cpp

```diff
@@ -5103,13 +5103,22 @@ void RichTextLabel::set_text(const String &p_bbcode) {
 	if (text == p_bbcode) {
 		return;
 	}
-
 	text = p_bbcode;
+	_apply_translated_bbcode(text);
+}
+
+void RichTextLabel::_apply_translated_bbcode(const String &p_bbcode) {
+	String xl_text;
+	if (is_auto_translating()) {
+		xl_text = tr(p_bbcode);
+	} else {
+		xl_text = p_bbcode;
+	}
```

Comment:
```suggestion
	String xl_text = atr(p_bbcode);
```
Same in other places.

Reply:
Thanks, I was wondering the difference between tr() and atr() when I saw it used in other places, now I get it :)